settings.gradle.kts

```diff
@@ -226,8 +229,13 @@ include(":system-tests:e2e-transfer-test:data-plane-postgresql")
 include(":system-tests:e2e-transfer-test:runner")
 include(":system-tests:management-api:management-api-test-runner")
 include(":system-tests:management-api:management-api-test-runtime")
+include(":system-tests:sts-api:sts-api-test-runner")
+include(":system-tests:sts-api:sts-api-test-runtime")
 
 include(":system-tests:telemetry:telemetry-test-runner")
 include(":system-tests:telemetry:telemetry-test-runtime")
 
 include(":version-catalog")
+include("extensions:common:iam:identity-trust:identity-trust-sts:identity-trust-sts-client-configuration")
+findProject(":extensions:common:iam:identity-trust:identity-trust-sts:identity-trust-sts-client-configuration")?.name =
+    "identity-trust-sts-client-configuration"
```

Comment:
what is the purpose of this assignment?

Reply:
No it's always intellj that does that, let me remove it thanks :)